src/lib/Sympa/Aliases.pm

```diff
@@ -135,6 +135,28 @@ sub check_new_listname {
             {new_listname => $listname});
     }
 
+    # Prevent to use prohibited listnames
+    my $regex = '';
+    if ($Conf::Conf{'prohibited_listnames_regex'}) {
+        $regex = lc($Conf::Conf{'prohibited_listnames_regex'});
+    }
```

Comment:
- Malformed regexps should be filtered, or Sympa might crash, for example,
    ``` perl
    $regex = eval(sprintf 'qr(%s)', $Conf::Conf{'prohibited_listnames_regex'}) // '';
    ```
  - `lc()`-ing regexp looks not good idea: E.g. `\W`, `\S` might be broken.

Reply:
I’ll use insensitive regex instead

Reply:
Done

Reply:
Sorry,
``` perl
            eval(sprintf 'qr(%s)', $Conf::Conf{'prohibited_listnames_regex'} // '');
```

Reply:
ping @ldidry

Reply:
I’ll do it this week.

Reply:
Sorry for the delay, I forgot. It’s done.